### PR TITLE
Add LDSTA to cime_developer test suite

### PR DESCRIPTION
**Pull request by mfdeakin-sandia**
See title

Test suite: cime_developer
Test baseline: 
Test namelist changes: 
Test status: BFB

### CODE REVIEW

File: scripts/lib/update_acme_tests.py [hunk 1]
```diff
@@ -53,7 +53,8 @@
                              "SMS.T42_T42.S",
                              "PRE.f19_f19.ADESP",
                              "PRE.f19_f19.ADESP_TEST",
-                             "MCC_P1.f19_g16_rx1.A")
+                             "MCC_P1.f19_g16_rx1.A",
+                             "LDSTA.ne4np4_oQU240.A")
```
Comment (jedwards4b) [inline scripts/lib/update_acme_tests.py]: I don't think that this grid is defined for cesm

Comment (mfdeakin-sandia) [reply scripts/lib/update_acme_tests.py]: Okay. What's the smallest grid e3sm and cesm support? Or does it affect A case runtimes and/or output file sizes?

Comment (jedwards4b) [reply scripts/lib/update_acme_tests.py]: f19_g16_rx1 is typical for testing

Comment (rljacob) [reply scripts/lib/update_acme_tests.py]: I think f19 is much higher res then ne4.  Try f45_g37_rx1

**Pull request merged**